<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="trafficNumEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="carriageNumEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="seatNumEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="departureEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="destinationEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 仿宋 Std R</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>COVID-19 同乘接触者查询系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="trafficNum">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>100</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>车次/班次编号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="carriageNum">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>150</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>车厢号(仅火车可填)：</string>
    </property>
   </widget>
   <widget class="QLabel" name="seatNum">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>200</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>座位号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="departure">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>250</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>出发地：</string>
    </property>
   </widget>
   <widget class="QLabel" name="destination">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>300</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>目的地：</string>
    </property>
   </widget>
   <widget class="QLabel" name="date">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>106</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择出行日期</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>296</width>
      <height>236</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>520</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>480</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="riskLevelEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="riskLevel">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>350</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>风险等级：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>386</y>
      <width>521</width>
      <height>151</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>440</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
